pkg/instrumentors/bpf/github.com/gin-gonic/gin/bpf/probe.bpf.c

```diff
@@ -0,0 +1,110 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "arguments.h"
+#include "span_context.h"
+#include "go_context.h"
+
+char __license[] SEC("license") = "Dual MIT/GPL";
+
+#define PATH_MAX_LEN 100
+#define METHOD_MAX_LEN 6 // Longer method: DELETE
+#define MAX_CONCURRENT 50
+
+struct http_request_t
+{
+    u64 start_time;
+    u64 end_time;
+    char method[METHOD_MAX_LEN];
+    char path[PATH_MAX_LEN];
+    struct span_context sc;
+};
+
+// map key: pointer to the goroutine that handles the request
+struct
+{
```

Comment:
Can we remain consistent and keep `{` on the same line as `struct` to match [existing](https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/3e35cba7b142ad98f14b02cc2bba5bfbfef2466d/pkg/instrumentors/bpf/github.com/gorilla/mux/bpf/probe.bpf.c#L24)?